main.tex

```diff
@@ -251,9 +251,11 @@ \subsection*{Functional variation}
 %
 The \texttt{L995F} allele at high frequency in populations of both species from West, Central and Southern Africa, and the \texttt{L995S} allele at high frequency among \textit{An. gambiae} populations from Central and East Africa (Table \ref{table:variants_missense}; \cite{Ag1000gConsortium2017}).
 %
-All haplotypes carrying L995F or L995S have evidence for strong recent positive selection \cite{Ag1000gConsortium2017}.
+All haplotypes carrying \texttt{L995F} or \texttt{L995S} have evidence for strong recent positive selection \cite{Ag1000gConsortium2017}.
 %
-Both alleles were present in populations sampled from Cameroon and Gabon, including some individuals with a hybrid \texttt{L995F/S} genotype. Within these populations, the \texttt{L995F} and \texttt{L995S} alleles were (@@TODO were not?) in Hardy-Weinberg equilibrium (P=@@), thus there does not (@@does?) appear to be selection against hybrids.
+Both alleles were present in populations sampled from Cameroon and Gabon, including some individuals with a hybrid \texttt{L995F/S} genotype.
+%
+In Cameroon these alleles were in Hardy Weinberg equilibrium ($x^{2}$ = 0.02, p > 0.05), thus there does not appear to be selection for or against carriers of both alleles, however in Gabon, they were not in equilibrium ($x^{2}$ = 8.96, p < 0.005), with an excess of hetrozygotes suggesting a fitness advantage to mosquitoes carrying both alleles in this region.
```

Comment:
Grammar nit, need semicolon or full stop new sentence before "however". E.g., "... of both alleles. However, in Gabon they were ..."

Also s/hetrozygotes/heterozygotes/